### PR TITLE
fix: reconnect sc event loop

### PR DESCRIPTION
**Pull request by hunjixin**
### Review Type Requested (choose one):
- [ ] **Glance** - superficial check (from domain experts)
- [ ] **Logic** - thorough check (from everybody doing review)

### Summary
Provide a one line summary and link to any relevant references

### Task/Issue reference

Closes: add_link_here

### Details (optional)
Add any additional details that might help Code Reviewers digest this PR

### CODE REVIEW

**Comment by hunjixin** [comment]
https://github.com/Lilypad-Tech/internal/issues/165

File: pkg/http/utils.go [hunk 1]
```diff
@@ -324,8 +323,8 @@ func GetRequestBuffer(
 
 func GenericJSONPostClient(url string, json string) (*http.Response, error) {
 	data := []byte(json)
-	client := &http.Client{Timeout: time.Second * 1}
-	req, err := http.NewRequest("POST", url, bytes.NewBuffer(data))
+	client := newRetryClient()
+	req, err := retryablehttp.NewRequest("POST", url, bytes.NewBuffer(data))
```
Comment (alvin-reyes) [inline pkg/http/utils.go:327]: why didn't we use the `client` that was initialize at 326?

File: pkg/web3/events_pow.go [hunk 2]
```diff
@@ -36,7 +37,7 @@ func (s *PowEventChannels) Start(
 
 	connectnewPowRoundSub := func() (event.Subscription, error) {
 		log.Debug().
-			Str("jobcreator->connect", "newPowRound").
+			Str("pow->connect", "newPowRound").
 			Msgf("")
```
Comment (alvin-reyes) [inline pkg/web3/events_pow.go]: Please set a message here on `Msgf`

File: pkg/http/utils.go (hunk 1 above)
Comment (hunjixin) [reply pkg/http/utils.go:327]: some node  report timeout here, so add a retry here

**Review by alvin-reyes — APPROVED** [review]
LGTM.

On the next PRs, please make sure to add description, verification step and what issues the PR resolves.

**Pull request merged**